lib/ui/puyo/game-duel.js

```diff
@@ -40,6 +40,17 @@ export default class PuyoDuelGame extends PuyoGameBase {
     } else {
       this.playerGrid.installEventListeners();
     }
+
+    window.setInterval(() => {
```

Comment:
This interval needs to be stopped when the game is stopped and user returns to lobby.